pkg/cli/config/config.go

```diff
@@ -86,7 +87,18 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	bpConfig.AssumeInitialArn = viper.GetString("assume-initial-arn")
 
 	// proxyURL is optional
-	proxyURL := viper.GetString("proxy-url")
+	proxyInConfigFile := viper.Get("proxy-url")
```

Comment:
If we use `viper.GetSlice` here, we can get a typed value back from the config instead of `any`. Then `getConfigProxyList` should be able to go away entirely.

Reply:
Fixed. Thanks for this suggestion. Reduced code by a lot.